src/components/UpcomingEvents/index.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState, useEffect } from 'react';
+import './styles.css';
+import { charityAPI } from '../../clients';
+// Heading global
+import Heading from '../Heading'
+// Date Package
+import {parseISO} from 'date-fns';
+// Images
+import bg_1 from './img/bg_1.png';
+
+// Some Other Components
+export const BackgroundImage = () => (
+  <div className="upcoming-events__bg-image absolute w-2/5">
```

Comment:
we can consider this in the future, I know you are not the original developer for this component, but I think it has some room for improvement (like having it as a background for the outer section or something, not sure)